cli/azd/pkg/devcentersdk/permissions_request_builder.go

```diff
@@ -33,10 +36,20 @@ func (c *PermissionListRequestBuilder) Get(ctx context.Context) ([]*armauthoriza
 		return nil, err
 	}
 
-	options := azsdk.NewClientOptionsBuilderFactory(c.client.options.Transport, "azd", c.client.cloud).
-		NewClientOptionsBuilder().
-		WithPerCallPolicy(azsdk.NewMsCorrelationPolicy()).
-		BuildArmClientOptions()
+	options := &arm.ClientOptions{
+		ClientOptions: azcore.ClientOptions{
+			PerCallPolicies: []policy.Policy{
+				azsdk.NewMsCorrelationPolicy(),
+				azsdk.NewUserAgentPolicy("azd"),
```

Comment:
It might be better to have this be `azsdk.NewUserAgentPolicy(internal.UserAgent()` so we get our version number in the user agent string, but what we have here matches the behavior of what we had before.